### PR TITLE
XAMARIN-43418 : UG content for OpenText and CloseText in SfBackdropPage.

### PR DESCRIPTION
**Pull request by BalachandarSelvam**
Hi Team,

To include UG content for OpenText and CloseText in SfBackdropPage.

Regards,
Balachandar.

### CODE REVIEW

**Comment by BalachandarSelvam** [comment]
rebuild

File: Xamarin/Backdrop-Page/Header-Configuration.md [hunk 1]
```diff
@@ -88,3 +88,45 @@ namespace BackdropGettingStarted
 
  ![Open Custom icons](Header-Configuration-images/Settings.png)
  
+## Icon text customization
+
+You can customize the icon text in the navigation header by setting the OpenText and CloseText properties in SfBackdropPage.
```
Comment (vigneshrameshsync) [inline Xamarin/Backdrop-Page/Header-Configuration.md:93]: Provide APIs as a link

File: Xamarin/Backdrop-Page/Header-Configuration.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Header Configuration with Syncfusion Backdrop Page
-description: How to configure a header in backdrop page.
+description: This section explains how to configure a header on the backdrop page by setting icons and also customizing the text for open and close icons.
```
Comment (vigneshrameshsync) [inline Xamarin/Backdrop-Page/Header-Configuration.md:4]: Change this by referring the getting started page.

**Review by vigneshrameshsync — CHANGES_REQUESTED** [review]

File: Xamarin/Backdrop-Page/Header-Configuration.md (hunk 2 above)
Comment (BalachandarSelvam) [reply Xamarin/Backdrop-Page/Header-Configuration.md]: Changed the content as per getting started content.

File: Xamarin/Backdrop-Page/Header-Configuration.md (hunk 1 above)
Comment (BalachandarSelvam) [reply Xamarin/Backdrop-Page/Header-Configuration.md]: Provided as APIs.

**Review by vigneshrameshsync — APPROVED** [review]
The content is fine.

**Pull request merged**